addon/templates/components/frost-navigation.hbs

```diff
@@ -13,8 +14,9 @@
       }}
     {{/if}}
   {{/each}}
-  {{#yield-slot 'actions'}}
-    {{yield}}
-  {{/yield-slot}}
+
+  <!-- Actions -->
+  {{component actions-container}}
```

Comment:
Can we alias this as `userMenu` instead of `actions-container`?  We can still keep a generic actions area, but the user should be required as the last entry on the right